Makefile

```diff
@@ -159,7 +159,7 @@ run: manifests generate ## Run a controller from your host.
 
 .PHONY: docker-build
 docker-build: ## Build docker image with the manager.
-	docker buildx build --load --platform linux/${ARCH} --build-arg OPERATOR_VERSION -t ${IMG} .
+	docker buildx build --load --platform linux/${ARCH} --build-arg OPERATOR_VERSION --build-arg TEMPO_VERSION -t ${IMG} .
```

Comment:
afaics we don't need to pass `OPERATOR_VERSION` or `TEMPO_VERSION` here, because in the Dockerfile we run `make build`, and the Makefile contains the versions (hardcoded).

Reply:
Ah, yes, but this is only required for testing, right? Anyway, still useful :+1: